test/distributed/fsdp/test_wrap.py

```diff
@@ -462,6 +462,17 @@ def test_module_wrap_policy(self):
         )
         self._test_transformer_wrapping(auto_wrap_policy)
 
+    @unittest.skipIf(not TEST_MULTIGPU, "Requires at least 2 GPUs")
```

Comment:
I see that the test file uses `@skip_if_lt_x_gpu(2)`. I was curious why you are using this different decorator.

Reply:
I am using the same decorator as the prior unit test method which I used as the basis of this new unit test method. This decorator is also used by other unit test methods in the same test case class.

https://github.com/pytorch/pytorch/blob/5cf481d1acb9f2dafd9687198c665ead112057bd/test/distributed/fsdp/test_wrap.py#L457-L463

Reply:
Ah good call! I missed this.

Your way makes sense to me :)